packages/lib-panoptes-js/panoptes.js

```diff
@@ -0,0 +1,67 @@
+const superagent = require('superagent');
+const config = require('./config');
+
+function handleErrors(error) {
+  if (console) console.error(error);
+
+  return error;
+}
+
+// TODO: Write auth client
+function get(endpoint, query, host) {
+  const apiHost = host || config.host;
+  const request = superagent.get(`${apiHost}${endpoint}`)
+    .set('Content-Type', 'application/json')
+    .set('Accept', 'application/vnd.api+json; version=1')
+  // .set('Authorization', apiClient.headers.Authorization);
+  if (query && query.length > 0) {
+    query.forEach((object) => {
+      if (typeof object === 'object') request.query(obj);
+      if (typeof object !== 'object') throw new Error('Query must be an object');
+    });
+  } else {
+    throw new Error('Queries must be in an array or array is empty');
+  }
+  return request.then(response => response).catch(handleErrors);
```

Comment:
I don't think this `.then` is actually needed if we're not modifying the response.